indico/modules/events/registration/schemas.py

```diff
@@ -0,0 +1,20 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from marshmallow.fields import String
+
+from indico.core.marshmallow import mm
+
+
+class RegistrationFormSchema(mm.ModelSchema):
```

Comment:
I'd rename this to `RegistrationFormPrincipalSchema` since it's very basic and if we ever add a more complete schema we don't want to suddenly start returning more details from the existing RH.

Reply:
Good idea